### DISCUSSION TITLE
Getting several errors on the website

### DISCUSSION THREAD

**Discussion by namas191297** [Budibase/budibase#7394] [Help] [answered]
Hi,

I recently created a few apps on the builder a few days ago but I constantly keep getting errors when trying to open or edit any of the applications today. The error is "Error initialising app: [object Object]" when I try to open an existing application or "Error creating app" when trying to create a new app. I also tried creating a different account but I'm getting the same problem. Any help would be appreciated.

Many thanks.
![errors](https://user-images.githubusercontent.com/42274709/185819401-99cff690-ca10-4066-84bb-fd1853e8269a.png)

**Comment by shogunpurple** [comment, marked as answer]
Hi @namas191297 - more information here https://github.com/Budibase/budibase/issues/7395#issuecomment-1221994154
